remote-secured/src/main/java/org/infinispan/tutorial/simple/remote/InfinispanRemoteSecured.java

```diff
@@ -24,8 +24,8 @@ public static void main(String[] args) {
 
       //Configure the security properties
       builder.security().authentication()
-              .username("Titus Bramble")
-              .password("Shambles")
+              .username("admin")
+              .password("pass")
```

Comment:
password needs to be updated.
it would be nice to have the same credentials in spring-boot tutorial as well.

Reply:
upgraded